### PR TITLE
Adds instructions for installing the latest ppb code.

### PR DESCRIPTION
**Pull request by pathunstrom**
Just adding these instructions so we can point to them when we want folks to be testing the latest stuff.

### CODE REVIEW

File: docs/howto/use-latest.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+===========================================================
+
+Sometimes you want to try the latest features that have been developed for
+ppb before they have a formal release on pypi. You have two options: provide
+a git uri to the ppb canon branch or use a dev release from test.pypi.org.
+
+With pip Directly
+----------------------------------------
+
+To install with git via pip, enter the following command into your shell::
```
Comment (AstraLuma) [inline docs/howto/use-latest.rst]:
```suggestion
To install with pip directly from github, enter the following command into your shell::
```

File: docs/howto/use-latest.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+To install using a requirements.txt file and the current git code, put
+``git+https://github.com/ppb/pursuedpybear.git`` as your requirement line
+then invoke pip as normal::
+
+   pip install -r requirements.txt
+
+To install from test.pypi.org with a requirements file, put ``ppb`` in your
+requirements file as normal, then invoke pip as so::
+
+   pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple --pre -r requirements.txt
```
Comment (AstraLuma) [inline docs/howto/use-latest.rst]: I'm pretty sure you can put at least some of these arguments into `requirements.txt`

**Review by AstraLuma — CHANGES_REQUESTED** [review]

File: docs/howto/use-latest.rst (hunk 2 above)
Comment (pathunstrom) [reply docs/howto/use-latest.rst]: Yeah, I was finding that after I opened this.

Want me to do that research and testing, or can I just open a ticket and do it as a follow up?

Comment (AstraLuma) [reply docs/howto/use-latest.rst]: Yeah, it can be a follow up if it's not easy testing.

Comment (pathunstrom) [reply docs/howto/use-latest.rst]: Did the testing, this is ready to go.

**Review by AstraLuma — APPROVED** [review]

**Comment by pathunstrom** [comment]
bors r=astraluma

**Comment by AstraLuma** [comment]
bors retry

**Pull request merged**